.github/workflows/pypi-publish.yml

```diff
@@ -5,24 +5,58 @@ name: Publish Python 🐍 distributions 📦 to PyPI
 
 on:
   release:
+    # • `released` reacts to stable releases
+    # • `prereleased` reacts to pre-releases
+    # • `published` reacts to both
     types: [published]
 
 jobs:
-  build-n-publish:
+  build:
+    name: Build distribution 📦
     runs-on: ubuntu-latest
-    permissions:
-      id-token: write # for trusted publishing
 
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-python@v4
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
+
+      - uses: actions/setup-python@v5
+
+      - name: Get history and tags for SCM versioning to work
+        run: |
+          git fetch --prune --unshallow
+          git fetch --depth=1 origin +refs/tags/*:refs/tags/*
```

Comment:
How come it worked until now without this?

Reply:
have no clue, maybe it was a change in the way github action checkouts the source? I could have a look how we got to this point, but I also think that we started without this and it got added at some point

Reply:
https://github.com/packit/packit/commit/bf36c82d7fb511b3a8c573ff3278c350317edfa7

Reply:
weren't the scm tools dropped since then? :eyes: if it's related to setuptools_scm

Reply:
setuptools_scm is still around (https://github.com/packit/specfile/blob/875f03eda8abb51e7211bd1de5c11bdb038cf03d/setup.py#L15, https://github.com/packit/specfile/blob/875f03eda8abb51e7211bd1de5c11bdb038cf03d/pyproject.toml#L9).